lms/djangoapps/instructor/views/api.py

```diff
@@ -277,6 +277,31 @@ def wrapped(request, course_id):  # pylint: disable=missing-docstring
     return wrapped
 
 
+def require_finance_admin(func):
```

Comment:
added the tests for the decorator https://github.com/edx/edx-platform/pull/7992/files#diff-1dcea18a471b3b764091199f6d021480R2051